play/src/front/Phaser/Game/MapEditor/AreasPropertiesListener.ts

```diff
@@ -550,23 +550,11 @@ export class AreasPropertiesListener {
                 jitsiUrl = answer.url;
             }
 
-            let domain = jitsiUrl || JITSI_URL;
+            const domain = jitsiUrl || JITSI_URL;
             if (domain === undefined) {
                 throw new Error("Missing JITSI_URL environment variable or jitsiUrl parameter in the map.");
             }
 
-            let domainWithoutProtocol = domain;
-            if (domain.substring(0, 7) !== "http://" && domain.substring(0, 8) !== "https://") {
-                domainWithoutProtocol = domain;
-                domain = `${location.protocol}//${domain}`;
-            } else {
-                if (domain.startsWith("http://")) {
-                    domainWithoutProtocol = domain.substring(7);
-                } else {
-                    domainWithoutProtocol = domain.substring(8);
-                }
-            }
-
```

Comment:
This is important.

JITSI_URL can contain "https://" or not. (it depends on the setups of the people who deployed WA).


Historically, we accepted the 2 possibilities so we must continue to do so.

This means we should rename "domain" to "jitsiUrl" and make sure "jitsiUrl" starts with a protocol (if it does not, we need to prepend jitsiUrl with https://)